diesel_derives/src/sql_function.rs

```diff
@@ -392,3 +452,21 @@ impl ToTokens for StrictFnArg {
         self.name.to_tokens(tokens);
     }
 }
+
+fn is_sqlite_return_type(ty: &syn::Type) -> bool {
```

Comment:
As we want to use that functions on sql functions with arguments to check the type of the argument too, I would prefer to just name it `is_sqlite_type`.

Reply:
Fixed.